src/FilterProviderInterface.php

```diff
@@ -7,6 +7,8 @@
 /**
  * Implement this interface within Module classes to indicate that your module
  * provides filter configuration for the FilterPluginManager.
+ *
+ * @deprecated Since 2.40.0 This interface will be removed in version 3.0 without replacement
```

Comment:
Must also be listed in the migration guide.

Reply:
Handled in #205